README.md

```diff
@@ -232,6 +563,10 @@ In terms of metadata this use case would likely need:
 - System and architecture details for where it was built
 - System and architecture details for where it has been successfully tested/run
 
+> shouldn't these come from the package manager? Seems like you could have different properties for
```

Comment:
I think there are enough containers created in the wild that you'd want to be able to derive these facts from a scanner regardless of if a package manager was used. We'd need to do something like scan/trace the entrypoint to see what is hit, and then look at other environment variables, or a custom user request.